src/GogeToken.sol

```diff
@@ -596,346 +584,350 @@ contract DogeGaySon is ERC20, Ownable {
 
     event GasForProcessingUpdated(uint256 indexed newValue, uint256 indexed oldValue);
 
-    event SwapAndLiquify(
-        uint256 tokensSwapped,
-        uint256 ethReceived,
-        uint256 tokensIntoLiqudity
-    );
+    event RoyaltiesTransferred(address indexed wallet, uint256 amountEth);
```

Comment:
Could just be amount `rather` than `amountEth`.

Reply:
I'm gonna stick with the Eth/Weth terminology because typically tokens on the EVM still use Eth verbage (though sometimes misleading), but I'd have to put a lot of energy changing this verbage sinc it's everywhere lol